addons/stock/models/stock_orderpoint.py

```diff
@@ -121,11 +133,31 @@ def write(self, vals):
                     raise UserError(_("Changing the company of this record is forbidden at this point, you should rather archive it and create a new one."))
         return super(Orderpoint, self).write(vals)
 
+    def _get_lead_days(self, rules):
+        delay = sum(rules.filtered(lambda r: r.action in ['pull', 'pull_push']).mapped('delay'))
+        delay_description = ''.join(['<tr><td>%s %s</td><td>+ %d day(s)</td></tr>' % (_('Delay on'), rule.name, rule.delay) for rule in rules if rule.action in ['pull', 'pull_push'] and rule.delay])
```

Comment:
```day(s)</td>``` is not translated

Reply:
now that i look a little more... why do you need all that? preparing html in a computed field is not clean + you'd need a t-raw at the other end

can't your javascript read all that?

Reply:
waiting for https://github.com/odoo/odoo/pull/41566